src/merge.py

```
@@ -501,16 +464,16 @@ def main():
     term.print_done()
     #endregion
 
-    print_step(TAG_CALENDAR_MERGE, term.TerminalColors.yellow.value + "processing source calendars" + term.TerminalColors.reset.value, one_liner=True)
+    print_step(TAG_CALENDAR_MERGE, term.TerminalColors.yellow.value + "processing source calendars" + term.TerminalColors.reset.value, True)
     while True:
         section = YAML_SECTION_SOURCE_CALENDAR.format(index=source_index)
         try:
             calendar_source = yaml_helper.get(section, YAML_SETTING_CALENDAR_SOURCE)
-        except YamlError:
-            print_step(TAG_CALENDAR_MERGE, term.TerminalColors.yellow.value + "no more source calendars to process" + term.TerminalColors.reset.value, one_liner=True)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)